src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -507,5 +507,18 @@ public static int countEmptyStrings(String... strings) {
      */
     public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
-    }  
-}
\ No newline at end of file
+    } 
+    
+    /**
+     * Returns a String array after removing white spaces preceding or
+     * succeeding any string in the input array.
```

Comment:
`leading and trailing`. 
You can even say `after applying the trim() method of the String class` (as it is a well known method).